src/parser/monk/mistweaver/modules/spells/EnvelopingMists.js

```diff
@@ -45,14 +61,15 @@ class EnvelopingMists extends Analyzer {
       return;
     }
 
-    if ((spellId === SPELLS.GUSTS_OF_MISTS.id) && (this.lastCastTarget === event.targetID)) {
+    if ((spellId === SPELLS.GUSTS_OF_MISTS.id) && (this.lastCastTarget === event.targetID) && this.numberToCount >0) {
       this.gustProc += 1;
       this.gustsHealing += (event.amount || 0) + (event.absorbed || 0);
+      this.numberToCount--;
```

Comment:
Another style item, but we have chosen to use `-= 1` to decrement. So just rework this is `this.numberToCount -= 1;`

Reply:
Will do Mr.BossMan